wazp/callbacks/metadata.py

```diff
@@ -346,9 +346,21 @@ def create_metadata_table_and_buttons(
                 ),
             )
 
+            # check for missing metadata files
+            check_missing_files_tooltip = dbc.Tooltip(
+                "Check which videos in the "
+                "video directory have a missing YAML file "
+                "and add a row for each of them. Note that "
+                "this won't generate any YAML files.",
+                target="add-rows-for-missing-button",
+            )
+
             generate_yaml_tooltip = dbc.Tooltip(
-                "Generate YAML files for each row in the selected spreadsheet "
-                "and save them in the videos directory",
+                "Generate YAML files from a selected spreadsheet. "
+                "Only rows with a corresponding video (or symlink) "
+                "in the video directory will be considered. "
+                "ATTENTION! This will overwrite any existing YAML files with "
+                "the same name!",
```

Comment:
```suggestion
                "Generate metadata files from a selected spreadsheet. "
                "Rows in the spreadsheet that do not correspond to a video will be ignored."
                "WARNING! This will overwrite any existing metadata files with "
                "the same name!",
```